docs/knowledgebase/getting-started/windows-users.md

```diff
@@ -43,34 +43,56 @@ If you do decide to try and use a Windows computer to _natively_ build Substrate
    rustup target add wasm32-unknown-unknown --toolchain nightly
    ```
 
-4. Install LLVM: https://releases.llvm.org/download.html
-
-5. Install OpenSSL with `vcpkg` using PowerShell:
-
-   ```bash
-   mkdir C:\Tools
-   cd C:\Tools
-   git clone https://github.com/Microsoft/vcpkg.git --depth=1
-   cd vcpkg
-   .\bootstrap-vcpkg.bat
-   .\vcpkg.exe install openssl:x64-windows-static
-   ```
-
-6. Add OpenSSL to your System Variables using PowerShell:
-
-   ```powershell
-   $env:OPENSSL_DIR = 'C:\Tools\vcpkg\installed\x64-windows-static'
-   $env:OPENSSL_STATIC = 'Yes'
-   [System.Environment]::SetEnvironmentVariable('OPENSSL_DIR', $env:OPENSSL_DIR, [System.EnvironmentVariableTarget]::User)
-   [System.Environment]::SetEnvironmentVariable('OPENSSL_STATIC', $env:OPENSSL_STATIC, [System.EnvironmentVariableTarget]::User)
-   ```
-
-7. Install `cmake`: https://cmake.org/download/
-
-8. Install `make`
-    - This can be done using Chocolatey. First you need to install the Chocolatey package manager: https://chocolatey.org/install
-    - Once Chocolatey installed you can install make:
-
-   ```
-   choco install make
-   ```
+4. Install dependencies:
+   
+   > **Important:** Windows systems have a restriction on path length which could lead to some issues. It is 
+   recommended to put your build folder closer to your root directory to avoid running into [these issues](https://github.com/substrate-developer-hub/substrate-node-template/issues/185).
```

Comment:
```suggestion
   recommended to put your build folder closer to your root directory to avoid running into
   [these issues](https://github.com/substrate-developer-hub/substrate-node-template/issues/185).
```

Reply:
line length